generic/tests/netperf.py

```diff
@@ -342,6 +364,9 @@ def _pin_vm_threads(vm, node):
             error_context.context("Change back server, client and host's mtu to %s"
                                   % mtu_default)
             mtu_set(mtu_default)
+        if params.get("client_physical_nic"):
+            cmd = 'ifconfig %s 0.0.0.0 down' % params.get("client_physical_nic")
+            ssh_cmd(client, cmd)
 
```

Comment:
@quanwenli it's a linux only cmd, may need to add a limitation.

Reply:
update, please help review. thanks